sample-submissions/llama_recipes/custom_dataset.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
This is a bit of a weird copyright notice, I worry this will infect the rest of the repo

Reply:
how about wget from gist instead?

Reply:
That sounds better

Reply:
done